install-tests/test_installation.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# build a simple project with clang
+
+set -eux
+VERSION=$1
+
+# bulid the demo project
+cd /build/sample_project
+export CC=clang-${VERSION}
```

Comment:
Also do
```
export CXX=clang++-${VERSION}
```
to set up the C++ compiler.

It's a preferred way to run the compiler for C++ using $(CXX). Otherwise, one has to specify extra flags to ensure the standard library is found and linked. If you ever want to test sanitizers on C++, it's going to become even more complicated to run `clang` instead of `clang++`